src/LSST/M1M3/SS/Settings/HardpointActuatorSettings.cpp

```diff
@@ -122,6 +122,24 @@ void HardpointActuatorSettings::load(YAML::Node doc) {
                             hardpointBreakawayFaultLow, hardpointBreakawayFaultHigh));
     }
 
+    inRangeAfterRaiseLoops = doc["InRangeAfterRaiseForce"].as<int>();
```

Comment:
```suggestion
    inRangeAfterRaiseLoops = doc["inRangeAfterRaiseLoops"].as<int>();
```
Also, are you sure you want to use camelCase on this one instead of CapsCase like all the others?